test/Dialect/Calyx/emit.mlir

```diff
@@ -14,7 +14,7 @@ module attributes {calyx.metadata = ["location1", "location2"], calyx.entrypoint
     calyx.control {}
   } {static = 1}
 
-  // CHECK-LABEL: component B<"toplevel"=1>(in: 1, @go go: 1, @clk clk: 1, @reset reset: 1) -> (out: 1, @done done: 1) {
+  // CHECK-LABEL: component B<"precious"=1, "toplevel"=1,>(in: 1, @go go: 1, @clk clk: 1, @reset reset: 1) -> (out: 1, @done done: 1) {
```

Comment:
We don't want a comma at the end here.

Reply:
Can't you just filter before, e.g., std::copy_if?

Reply:
This would be an ideal case for C++20 ranges but I think MLIR is still in 17 mode

Reply:
Or the llvm way is llvm::erase_if